framework/src/materials/GenericConstantMaterial.C

```diff
@@ -46,6 +46,12 @@ GenericConstantMaterial::GenericConstantMaterial(const InputParameters & paramet
 }
 
 void
+GenericConstantMaterial::initQpStatefulProperties()
```

Comment:
The suggestion is that this function should be never called because this material does not `getPropertyOld`.